### PR TITLE
doc: fixing python 3 bug in decoding example

### PR DESCRIPTION
**Pull request by choldgraf**
Was getting a floating point indexing error. Hold off merging for a sec - it seems like there are a few other errors along these lines in the codebase. LMK if just wrapping stuff in `int(` is an OK solution. The problem is that since Python 3 returns a float if you do, e.g., `10 / 2`, then if you use this as an index it errors.

### CODE REVIEW

**Comment by kingjr** [comment]
use `a // b` instead

**Comment by choldgraf** [comment]
good call - I'll make this switch once I get the docs to finish building on my machine using python 3 :P

**Comment by choldgraf** [comment]
mmk, all the examples build for me now. for some reason one tutorial returns an error only when running with sphinx gallery but not when I run it on its own...don't think it's a python3 problem though.

File: examples/time_frequency/plot_source_space_time_frequency.py [hunk 1]
```diff
@@ -54,7 +54,7 @@
 stcs = source_band_induced_power(epochs, inverse_operator, bands, n_cycles=2,
                                  use_fft=False, n_jobs=1)
 
-for b, stc in stcs.iteritems():
+for b, stc in stcs.items():
```
Comment (jasmainak) [inline examples/time_frequency/plot_source_space_time_frequency.py]: do we have a convention that our examples are python 3 compatible but not python 2?

**Comment by Eric89GXL** [comment]
They should be compatible with both, and this looks like it is

File: examples/time_frequency/plot_source_space_time_frequency.py (hunk 1 above)
Comment (agramfort) [inline examples/time_frequency/plot_source_space_time_frequency.py]: they should be compatible with boths. It's the case now here.

**Comment by agramfort** [comment]
thx @choldgraf

**Pull request merged**